File: assets/samples/domain.availability/password/nodejs/domain.avail.js
var http = require('http');

var domain = 'example.com';
var password = 'Your domain availability api password';
var username = 'Your domain availability api username';

var url = 'http://www.whoisxmlapi.com/whoisserver/WhoisService?'
        + 'domainName=' + domain + '&username=' + username + '&password=' + password
        + '&cmd=GET_DN_AVAILABILITY' + '&getMode=DNS_AND_WHOIS';

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
}).end();

File: assets/samples/domain.availability/password/perl/domain.avail.pl
#!/usr/bin/perl

use LWP::Simple;

my $domain = 'example.com';
my $password = 'your domain availability api password';
my $username = 'your domain availability api username';

my $url = 'http://www.whoisxmlapi.com/whoisserver/WhoisService?'
        . "domainName=$domain&userName=$userName&password=$password"
        . "&cmd=GET_DN_AVAILABILITY" . "&getMode=DNS_AND_WHOIS";

print get($url);